wazo_call_logd/cel_interpretor.py

```diff
@@ -94,6 +94,55 @@ def _extract_call_log_destination_variables(extra: dict) -> dict:
     return extra_dict
 
 
+def _extract_pickup_target(appdata: str):
+    exten_pattern = r'[\w*#]+'
+    context_pattern = r'[\w-]+'
+    pattern = re.compile(
+        rf'(?P<exten>{exten_pattern})%(?P<context>{context_pattern})@PICKUPMARK'
+    )
+    match = pattern.match(appdata)
+    if match:
+        groups = match.groupdict()
+        return groups['exten'], groups['context']
+
+
+def _extract_pickup_info(cel: CEL) -> PickupInfo | None:
+    if cel.appname != 'Pickup':
+        return None
+    elif not cel.appdata:
+        logger.error('Missing expected pickup appdata for Pickup application event')
+        return PickupInfo(interceptor_channel=cel.channame)
```

Comment:
Is this a valid `PickupInfo`? Is `PickupInfo` consumed somewhere? I could not find where it is used.
Should we return `None` instead?